### PR TITLE
Mihailuca branch

### PR DESCRIPTION
**Pull request by lucamihai**
added Mihai Luca tasks

### CODE REVIEW

**Comment by precupstefan** [comment]
Hello kind sir

**Comment by precupstefan** [comment]
https://www.youtube.com/watch?v=dQw4w9WgXcQ

File: mihailuca-dir/mihailuca.txt [hunk 1]
```diff
@@ -0,0 +1,2 @@
+this file was created by Mihai Luca
+second commit
```
Comment (vladstanescu94) [inline mihailuca-dir/mihailuca.txt]: please add "reviewed by Vlad Stanescu"

**Review by vladstanescu94 — CHANGES_REQUESTED** [review]

**Review by precupstefan — APPROVED** [review]
In viata mea, in toata viata mea n-am vazut un cod mai urat decat asta

**Comment by precupstefan** [comment]
Eu i-am dat approve, da @vladstanescu94 vrea sa-i dea de lucru 😢

**Review by precupstefan — APPROVED** [review]

**Review by lucamihai — COMMENTED** [review]
a

**Review by vladstanescu94 — APPROVED** [review]

File: mihailuca-dir/mihailuca.txt (hunk 1 above)
Comment (steprp) [reply mihailuca-dir/mihailuca.txt]: O copiat de la mine

**Pull request merged**